### ISSUE TITLE
Changes to ConsIndShockModel in commit "9020084" have broken SolvingMicroDSOPs and possibly other applications

### ISSUE THREAD

**Issue by npalmer-professional** [econ-ark/HARK#175]
If you run the following under the current codebase as of today (most recent commit is [43952d1](https://github.com/econ-ark/HARK/commit/43952d127a432d473e2509582393af5119f9d363) at the time of this writing):

    import HARK.SolvingMicroDSOPs.StructEstimation as struct
    struct.main() 

You will get this error: 

    TypeError: unsupported operand type(s) for /: 'float' and 'NoneType'

This appears to be due to changes implemented in [this commit](https://github.com/econ-ark/HARK/commit/902008460fbb1d4c2986b10234645727f2629199). To check, you can pull the [previous commit](https://github.com/econ-ark/HARK/commit/4b340003d36c48d33c4e2e6adc7b25437a8cb7ee)  (PR#172) and the above example lines will run fine. 

I am pretty sure some of the whitespace change that shows up in the [possible problem commit](https://github.com/econ-ark/HARK/commit/902008460fbb1d4c2986b10234645727f2629199) are the reason for this.

I will look into this in the morning. 

Until then we should *not* update the HARK up on pip/conda, because those appear to be made using the last working [previous commit](https://github.com/econ-ark/HARK/commit/4b340003d36c48d33c4e2e6adc7b25437a8cb7ee)  (PR#172).

**Comment by mnwhite** [comment]
Was this addressed before the latest version was pushed to pip?

**Comment by mnwhite** [comment]
It looks like those commits were reverted, and might have been pushed in error.  I don't think they could be the cause of current errors?

**Comment by npalmer-professional** [comment]
Solved. Could not find a good way to clean up the history of my "ReplicateMake" branch, so did the following: 

Did a force reset for `npalmer-professional:master` to `econ-ark:master` for the most recent common commit after the reverts. Then created a new `ReplicateMake-Take3` branch, manually moved the changes to this new branch, and rebased/squashed `ReplicateMake-Take3` into `npalmer-professional:master`. 

Then I compared the histories between `econ-ark:master` and `npalmer-professional:master`  using 
    git log --pretty=oneline --graph --decorate --all
and everything looked good. I then checked the diff between `econ-ark:master` and `npalmer-professional:master`. Since both the histories and diffs looked as expected, I issued pull request #176 .
